ext/merge2std2/ranges-lib.tex

```diff
@@ -148,15 +148,16 @@
     template <class F, class I1, class I2 = I1>
     concept @\oldtxt{bool}@ IndirectStrictWeakOrder = @\seebelow@;
 
-    template <class> struct indirect_result@\oldtxt{_of}@;
+    template <class@\newtxt{, class...}@>
+    struct indirect_result@\oldtxt{_of}@ { };
```

Comment:
Markup?

Reply:
Better yet, just remove the definition. It doesn't need to be both here *and* in the detailed specification.

Reply:
... or just put the definition in the synopsis. After all, the definition is just `{ }`.

Reply:
That's probably how I would have written this in a green field, but with no real reason to prefer one style or the other I went with what was in R0: non-defining declaration here, definition in the detailed spec.

Reply:
No worries, I'll fix it in P1037.

Reply:
Fixed.